### DISCUSSION TITLE
Ability to specify Scoop Radius

### DISCUSSION THREAD

**Discussion by granthworth** [Le0Michine/FusionGridfinityGenerator#17] [Ideas]
In order to save filament, a smaller scoop radius could be calculated by entering a height unit or percentage of the total bin height. I have found myself generating bins with no scoop, deleting 3 bottom inside corner radii in each compartment, and then specifying my own scoop radius with a new fillet. No big deal if this is complicated to implement. Thanks for your hard work!

**Comment by Le0Michine** [comment]
Hi, as a quick solution I can suggest generating bin with scoop and then editing the radius in the timeline, that might be easier. I'll see if I can expose scoop radius, probably it's time to consider an extra settings section like "advance settings" or something to avoid amount of parameters getting out of control

**Comment by Le0Michine** [comment]
was added in this PR https://github.com/Le0Michine/FusionGridfinityGenerator/pull/27
